lib/timer.js

```diff
@@ -71,11 +80,12 @@ class Timer extends Component {
         return;
       }
       this.setState({remainingTime: remaining});
-    }, 1);
+    }, 100);
   }
 
   stop() {
     clearInterval(this.interval);
+    this.interval = null;
```

Comment:
Set `this.interval` to `null` so that we can prevent `start` from doing anything if `interval is not `null`, i.e. its already running